<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControleEstoque</class>
 <widget class="QDialog" name="ControleEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle de Estoque</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="logoHorizontalLayout">
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="minimumSize">
        <size>
         <width>264</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>264</width>
         <height>100</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/imagens/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>807</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabConsulta">
      <attribute name="title">
       <string>Consulta</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutConsulta">
       <item>
        <layout class="QHBoxLayout" name="filtrosLayoutConsulta">
         <item>
          <widget class="QLineEdit" name="buscaLineEdit">
           <property name="styleSheet">
            <string>
             QLineEdit {
               border: 2px solid #FF66B2;
               border-radius: 10px;
               padding: 8px;
               font-size: 14px;
               color: #FF66B2;
             }
             QLineEdit:focus {
               border: 2px solid #FF3399;
             }
            </string>
           </property>
           <property name="placeholderText">
            <string>Buscar por Nome, Marca ou Código de Barras...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="grupoComboBox">
           <property name="styleSheet">
            <string>
             QComboBox {
               border: 2px solid #FF66B2;
               border-radius: 10px;
               padding: 8px;
               font-size: 14px;
               color: #FF66B2;
             }
             QComboBox:focus {
               border: 2px solid #FF3399;
             }
             QComboBox::drop-down {
               border: none;
               background: transparent;
             }
             QComboBox::down-arrow {
               image: url(:/resources/arrow-down.png);
             }
            </string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tabelaProdutos">
         <property name="columnCount">
          <number>6</number>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Grupo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Marca</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qtd</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Preço</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="botoesConsultaLayout">
         <item>
          <widget class="QPushButton" name="btnVoltarConsulta">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>
             QPushButton {
               background-color: transparent;
               color: #DC3545;
               border-radius: 15px;
               border: 2px solid #DC3545;
               padding: 8px 15px;
               font-size: 16px;
             }
             QPushButton:hover {
               background-color: #DC3545;
               color: white;
               border: 2px solid #DC3545;
             }
            </string>
           </property>
           <property name="text">
            <string>Voltar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacerConsulta">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMovimentacao">
      <attribute name="title">
       <string>Movimentação</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutMovimentacao">
       <item>
        <layout class="QHBoxLayout" name="filtrosLayoutMovimentacao">
         <item>
          <widget class="QLineEdit" name="buscaMovimentacaoLineEdit">
           <property name="styleSheet">
            <string>
             QLineEdit {
               border: 2px solid #FF66B2;
               border-radius: 10px;
               padding: 8px;
               font-size: 14px;
               color: #FF66B2;
             }
             QLineEdit:focus {
               border: 2px solid #FF3399;
             }
            </string>
           </property>
           <property name="placeholderText">
            <string>Buscar por Nome, Marca ou Código de Barras...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="grupoMovimentacaoComboBox">
           <property name="styleSheet">
            <string>
             QComboBox {
               border: 2px solid #FF66B2;
               border-radius: 10px;
               padding: 8px;
               font-size: 14px;
               color: #FF66B2;
             }
             QComboBox:focus {
               border: 2px solid #FF3399;
             }
             QComboBox::drop-down {
               border: none;
               background: transparent;
             }
             QComboBox::down-arrow {
               image: url(:/resources/arrow-down.png);
             }
            </string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="movimentacaoTableWidget">
         <property name="minimumSize">
          <size>
           <width>807</width>
           <height>280</height>
          </size>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Grupo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Marca</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qtd Atual</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Preço</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qtd Entrada</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qtd Saída</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="botoesMovimentacaoLayout">
         <item>
          <widget class="QPushButton" name="btnVoltar">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>
             QPushButton {
               background-color: transparent;
               color: #DC3545;
               border-radius: 15px;
               border: 2px solid #DC3545;
               padding: 8px 15px;
               font-size: 16px;
             }
             QPushButton:hover {
               background-color: #DC3545;
               color: white;
               border: 2px solid #DC3545;
             }
            </string>
           </property>
           <property name="text">
            <string>Voltar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacerMovimentacaoBotoes">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="confirmarMovimentacaoButton">
           <property name="styleSheet">
            <string>
              QPushButton {
                background-color: #FF66B2;
                color: white;
                border-radius: 15px;
                border: 2px solid #FF3399;
                padding: 10px 20px;
                font-size: 16px;
              }
              QPushButton:hover {
                background-color: #FF3399;
                border-color: #FF66B2;
              }
            </string>
           </property>
           <property name="text">
            <string>Confirmar Movimentação</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="movimentacaoMsgLabel">
         <property name="styleSheet">
          <string>
            QLabel {
              font-weight: bold;
              font-size: 14px;
              color: #FF66B2;
            }
          </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
